File: show_me_the_pets_client/src/components/Pet.js
import React from 'react'

const Pet = ({ pet }) => {
  return(
    <tr>
      <td class="select-pet" onClick={() => this.showPet(pet.id)}>{pet.name}</td>
      <td>{pet.animal_type}</td>
      <td>{pet.color}</td>
      <td>{pet.breed}</td>
      <td>{pet.gender[0]}</td>
      <td>{pet.address}</td>
    </tr>
  )
}
export default Pet

File: show_me_the_pets_api/app/models/pet.rb
class Pet < ActiveRecord::Base
  # attr_accessor :id, :name, :animal_type, :gender, :breed, :color
  self.primary_key = "animal_id"

  def self.new_from_csv(row)
    elements = row.to_s.split(",")
    attrs = {
      animal_id: elements[0],
      name: elements[1],
      animal_type: elements[2],
      gender: elements[3],
      breed: elements[4],
      color: elements[5],
      address: elements[6]
    }
    Pet.create(attrs)
  end

end
